libbeat/processors/add_docker_metadata/add_docker_metadata.go

```diff
@@ -266,15 +267,30 @@ func (d *addDockerMetadata) getProcessCgroups(pid int) (map[string]string, error
 }
 
 // getContainerIDFromCgroups checks all of the processes' paths to see if any
-// of them are associated with Docker. Docker uses /docker/<CID> when
-// naming cgroups and we use this to determine the container ID. If no container
-// ID is found then an empty string is returned.
-func getContainerIDFromCgroups(cgroups map[string]string) string {
+// of them are associated with Docker. Docker uses /docker/<CID> and
+// Kubernetes, with Docker as a container runtime, uses
+// /kubepods/<class>/<podID>/<cID> when naming cgroups.
+// We use this to determine the container ID.
```

Comment:
I think this will not work in cases of having more than one containers for a Pod. For instance, `pause` containers will be found under `/kubepods/<class>/<podID>` along with the main containers,  and the structure would look like this:
1. `/kubepods/<class>/<podID>/<cID1>`
2. `/kubepods/<class>/<podID>/<cID2>`
In this case how we identify which cID we are interested in?